### PR TITLE
Fix grammar in check-if-dockershim-removal-affects-you page

### PR DESCRIPTION
**Pull request by wwalters12**
Change `Kubernetes allows to use Docker` to  `Kubernetes allows the use of Docker`

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
I don't care either way. Both wordings sounds ok to me, but I'm not a native speaker. Looks like it falls under the [trivial edit policy](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits)

**Comment by wwalters12** [comment]
> I don't care either way. Both wordings sounds ok to me, but I'm not a native speaker. Looks like it falls under the [trivial edit policy](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits)

Both wordings convey the same idea, but `Kubernetes allows to use Docker` sounds incorrect to a native speaker.

I reviewed the rest of the file and found a few additional grammatical errors, but I understand if this still falls under the trivial edit policy.

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ weight: 50
 
 <!-- overview -->
 
-The `dockershim` component of Kubernetes allows to use Docker as a Kubernetes's
+The `dockershim` component of Kubernetes allows the use of Docker as a Kubernetes's
```
Comment (bradtopol) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you.md:11]: Great job! All the edits look great.

**Comment by bradtopol** [comment]
Thank you @wwalters12 for your contribution and taking the extra time to address multiple issues on the page.  The edits looks great and rendered correctly in the preview. Great job!
/lgtm
/approve

**Pull request merged**